Pylint errors

<!---
Use this issue template for reporting a bug.

Before opening a new issue, please verify the issue you're about to submit isn't a duplicate.
--->

### Summary

`pylint` reports two errors.

### Steps to reproduce
```
pylint -E sal/ 
```
### What is the current *bug* behavior?
```
 ************* Module sal.dataclass.signal.error.asymmetric
sal/dataclass/signal/error/asymmetric.py:112:29: E1306: Not enough arguments for format string (too-few-format-args)
************* Module sal.core.tests.test_path
sal/core/tests/test_path.py:3:0: E0401: Unable to import 'jetserver.jetvfs.path' (import-error)
```

### What is the expected *correct* behavior?

No errors

### Possible fixes
- Add an argument to format string
- Remove the invalid import

--- a/sal/core/tests/test_path.py
+++ b/sal/core/tests/test_path.py
@@ -1,6 +1,6 @@
 import unittest
 
-import jetserver.jetvfs.path as pth
+from sal.core import path as pth
 
 VALID_SEGMENTS = [
     "abcdefghijklmnopqrstuvwxyz._-01234567890",

--- a/sal/dataclass/signal/error/asymmetric.py
+++ b/sal/dataclass/signal/error/asymmetric.py
@@ -109,7 +109,7 @@ class AsymmetricArrayError(Error):
             raise ValueError('The dictionary does not contain a serialised asymmetrical array error class.')
 
         if d['lower'].dtype != d['upper'].dtype:
-            raise ValueError('Cannot deserialise, lower and upper data types are inconsistent (lower: {}, upper: {})'.format(d['lower'].dtype), dtype=d['upper'].dtype)
+            raise ValueError('Cannot deserialise, lower and upper data types are inconsistent (lower: {}, upper: {})'.format(d['lower'].dtype, d['upper'].dtype))
 
         c = cls(
             lower=d['lower'],

